### PR TITLE
docs: configure storage provider

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

A recipe to show how to setup azure storage.

## Testing

image

## Checklist

- [ ] `.changset-staged`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: docs/docs/recipes/configure-storage-providers/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 10
+---
+
+# 💾 Configure storage providers
+
+To store user uploaded files, Logto uses a storage provider. Logto supports multiple storage providers, including AWS S3, Azure Storage. This recipe will show you how to configure a storage provider for Logto.
+
+The configuration is stored in DB's `logto_configs` table, but it is recommended to use the CLI to configure the storage provider.
```
Comment (gao-sun) [inline docs/docs/recipes/configure-storage-providers/README.md:9]: should be `systems`? since it's a global config but `logto_configs` is per-tenant

Comment (wangsijie) [reply docs/docs/recipes/configure-storage-providers/README.md]: updated, the core code need to update as well

File: docs/docs/recipes/configure-storage-providers/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (gao-sun) [inline docs/docs/recipes/configure-storage-providers/README.md:1]: will we leave this open until it's complete?

**Review by darcyYe — APPROVED** [review]

File: docs/docs/recipes/configure-storage-providers/README.md (hunk 2 above)
Comment (wangsijie) [reply docs/docs/recipes/configure-storage-providers/README.md:1]: It's ready now

**Pull request merged**